### DISCUSSION TITLE
AntDeploy支持哪几个vs版本

### DISCUSSION THREAD

**Discussion by oldyangsir** [yuzd/AntDeploy#35] [Q&A] [answered]
AntDeploy vs插件，具体支持哪些vs版本

**Comment by yuzd** [comment, marked as answer]
vs2013~vs2018的插件下载地址：https://marketplace.visualstudio.com/items?itemName=nainaigu.AntDeploy

vs2019插件下载地址：https://marketplace.visualstudio.com/items?itemName=nainaigu.AntDeployVsix

vs2022插件下载地址： https://marketplace.visualstudio.com/items?itemName=nainaigu.AntDeploy2022

另外也有独立软件版本，参考群(qq:488312978)文件
